apis/fluent-en_GB/atrium-api-fluent-en_GB-jvm/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/PathAssertionSamples.kt

```diff
@@ -0,0 +1,30 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.toBeASymbolicLink
+import ch.tutteli.atrium.api.verbs.internal.expect
+import java.nio.file.Files
+import kotlin.test.Test
+
+class PathAssertionSamples {
+
+    private val tempDir = Files.createTempDirectory("PathAssertionSamples")
+
+    @Test
+    fun isASymbolicLink() {
+        val target = tempDir.resolve("target")
+        val link = Files.createSymbolicLink(tempDir.resolve("link"), target)
+
+        // Passes, as subject `link` is a symbolic link
+        expect(link).toBeASymbolicLink()
+    }
+
+    @Test
+    fun isNotASymbolicLink() {
+        val path = tempDir.resolve("somePath")
+
+        // Fails, as subject `path` is a not a symbolic link
```

Comment:
It's correct but I think the example would be better if the file actually exists. I suggest you create a regular file at that place